object/service.proto

```diff
@@ -326,10 +326,14 @@ message SearchRequest {
     //
     // * $Object:ROOT \
     //   With the `value` set to `true` checks if an object is a top object in a
-    //   split hierarchy. With other values returns non-root objects.
+    //   split hierarchy. Works only with `REGULAR` type objects, including
+    //   virtual split objects. With other values returns non-root objects of
+    //   `REGULAR` type.
     // * $Object:LEAF \
     //   With the `value` set to `true` checks if an object is a leaf in a split
-    //   hierarchy. With other values returns non-leaf objects.
+    //   hierarchy. Works with all types of objects but only physically stored
+    //   on the node, without virtual split objects. With other values returns
+    //   non-leaf objects.
```

Comment:
`LEAF` filter is not related to split hierarchy at all. 

With the `value` set to `true` returns objects that physically stored  on the node.
With other value returns virtual objects.